src/components/Calculator.js

```diff
@@ -1,33 +1,60 @@
-function Calculator() {
-  return (
-    <div className="container">
-      <form>
-        <input className="output" type="text" />
-      </form>
+import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
-      <div className="btns">
-        <button type="button" className="grey">AC</button>
-        <button type="button" className="grey">+/-</button>
-        <button type="button" className="grey">%</button>
-        <button type="button" className="orange">&divide;</button>
-        <button type="button" className="grey">7</button>
-        <button type="button" className="grey">8</button>
-        <button type="button" className="grey">9</button>
-        <button type="button" className="orange">&times;</button>
-        <button type="button" className="grey">4</button>
-        <button type="button" className="grey">5</button>
-        <button type="button" className="grey">6</button>
-        <button type="button" className="orange">&minus;</button>
-        <button type="button" className="grey">1</button>
-        <button type="button" className="grey">2</button>
-        <button type="button" className="grey">3</button>
-        <button type="button" className="orange">+</button>
-        <button type="button" className="grey span-2">0</button>
-        <button type="button" className="grey">.</button>
-        <button type="button" className="orange">=</button>
-      </div>
-    </div>
-  );
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      total: 0,
+      next: null,
+      operator: null,
+    };
+  }
+
+    handleClick = (event) => {
+      const result = (calculate(this.state, event.target.textContent));
+      this.setState(result);
+    }
+
+    resultDisplay = (event) => {
+      this.setState({ total: event.target.textContent });
+    }
+
+    render() {
+      const { total, next, operation } = this.state;
+      return (
+        <div className="container">
+          <div className="output" onChange={this.resultDisplay}>
+            {total}
+            {operation}
+            {next}
+          </div>
```

Comment:
- [ ] For this milestone, you're required to handle state management with the `useState` hook.
Below is a sample of how your component should look like:
```javascript
// ./src/components/Calculator.js

import React, { useState } from 'react';
import calculate from ../logic; // Import math logic

function Calculator() {
    const [total, setTotal] = useState(0);
    return {
      <div>
       // Calculator app 
      </div>
    };
}
```
**Your Project requirement**

<img width="1101" alt="Screenshot 2023-02-16 at 2 04 15 PM" src="https://user-images.githubusercontent.com/50941074/219387626-09c354dc-7a42-4df2-8497-02c0d4182bed.png">

<img width="620" alt="Screenshot 2023-02-16 at 2 06 25 PM" src="https://user-images.githubusercontent.com/50941074/219387826-60399f8c-7c06-4f85-98f8-883994a5e6b9.png">

- [ ] Kindly make use of the react useState hook 👍🏽

Reply:
Hello @AmaduKamara, 

Thank you for your review but Please Sir using useState Hook is the fourth project of the week which is Hooks project refactoring from class based component to functional component and not for the third day project. That was actually the project requirement.

Reply:
resolved! ✔️